pyiron_base/jobs/job/template.py

```diff
@@ -49,3 +50,17 @@ def __init__(self, project, job_name):
         super().__init__(project, job_name)
         self._python_only_job = True
         self._write_work_dir_warnings = False
+
+
+class PythonTemplateJobWithExecutor(PythonTemplateJob, JobWithExecutor):
```

Comment:
Do we really need a new class here, or can `PythonTemplateJob` just be expanded to _always_ inherit from `JobWithExecutor`? As far as I can tell the storage and computational overhead of `JobWithExecutor` is extremely minimal, so I'd just beef up the main class instead of proliferating new user-facing tools.

Reply:
I removed this class and implemented the functionality in the `GenericJob` class.